js/carousel.js

```diff
@@ -59,7 +60,7 @@
 
     this.options.interval
       && !this.paused
-      && (this.interval = setInterval($.proxy(this.next, this), this.options.interval))
+      && (this.interval = setInterval($.proxy(this.next, this), this.duration = this.$element.find('.item.active').data('duration') || this.options.interval))
```

Comment:
Any particular reason for using `this.$element.find('.item.active')` instead of `this.$active` here? And also, why save the value. I don't see it being used anywhere else.